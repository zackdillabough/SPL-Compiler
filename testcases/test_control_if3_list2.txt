
a , b ;
{
input a;
input b;
IF  a <> b
{
	output b;
	output a;
	a = a + b; 
	output a;
	b = a + b;
	output b;
}
}
7 9
