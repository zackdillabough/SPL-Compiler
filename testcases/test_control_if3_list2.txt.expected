9 7 16 25 